Tests/Packet++Test/Tests/DoIpTests.cpp

```diff
@@ -0,0 +1,1402 @@
+#include "../TestDefinition.h"
+#include "../Utils/TestUtils.h"
+#include "EndianPortable.h"
+#include "Packet.h"
+#include "EthLayer.h"
+#include "IPv4Layer.h"
+#include "TcpLayer.h"
+#include "UdpLayer.h"
+#include "PayloadLayer.h"
+#include "SystemUtils.h"
+#include "PacketUtils.h"
+#include "DeprecationUtils.h"
+#include <memory>
+#include <array>
+#include "DoIpLayer.h"
+#include "GeneralUtils.h"
+
+// RoutingActivationRequest
+PTF_TEST_CASE(DoIpRoutingActivationRequestPacketParsing)
+{
+	// Dissect Routing Activation Request message
+	timeval time;
+	gettimeofday(&time, nullptr);
+
+	READ_FILE_AND_CREATE_PACKET(1, "PacketExamples/DoIpRoutingActivationRequestPacket.dat");
+
+	pcpp::Packet RoutingActivationRequest(&rawPacket1);
+	PTF_ASSERT_TRUE(RoutingActivationRequest.isPacketOfType(pcpp::IPv4));
+	PTF_ASSERT_TRUE(RoutingActivationRequest.isPacketOfType(pcpp::TCP));
+	PTF_ASSERT_TRUE(RoutingActivationRequest.isPacketOfType(pcpp::DOIP));
+
+	pcpp::TcpLayer* tcpLayer = RoutingActivationRequest.getLayerOfType<pcpp::TcpLayer>();
+	PTF_ASSERT_NOT_NULL(tcpLayer);
+
+	PTF_ASSERT_EQUAL(tcpLayer->getDstPort(), pcpp::DoIpPorts::TCP_UDP_PORT);
```

Comment:
Same here: testing the TCP/UDP ports isn't needed - we know the packet's port, so if we see the DoIP it means it was parsed correctly.

We can remove the port tests from the other tests also